sklearn/pipeline.py

```diff
@@ -623,6 +623,10 @@ def _pairwise(self):
         # check if first estimator expects pairwise input
         return getattr(self.steps[0][1], '_pairwise', False)
 
+    @property
+    def n_features_in_(self):
```

Comment:
How will you define this for FeatureUnion and ColumnTransformer?

Reply:
For column transformer, wouldn't that be the number of features of the original `X`? Especially if we implement option (1) from https://github.com/scikit-learn/scikit-learn/issues/14251#issuecomment-508199957, it would seem definitely logical.

Reply:
Yes I think the answer to the question is closely related to #14251 and one of the reasons I opened that issue.